sotodlib/mapmaking/demod_mapmaker.py

```diff
@@ -5,12 +5,14 @@
 you accumulate observations into the div and rhs maps. For examples
 how to use look at docstring of DemodMapmaker.
 """
-__all__ = ['DemodMapmaker','DemodSignal','DemodSignalMap']
-import numpy as np
-from pixell import enmap, utils, tilemap, bunch
+__all__ = ['DemodMapmaker','DemodSignal','DemodSignalMap','make_demod_map']
+import numpy as np, os
+from pixell import enmap, utils as putils, tilemap, bunch, mpi
 import so3g.proj
 
+from .. import core
 from .. import coords
+from .. import site_pipeline
```

Comment:
I don't really like that library code imports things from `site_pipeline`.  That module is sort of special.  For now can you import that submodule from inside your driver function?

The load_or_preproc function, or whatever it's called, should probably move to library level... that's beyond the scope of this PR though.

Reply:
Ok. I agree, probably this function should be in core next to get_obs. @msilvafe I will create an issue to remind us